packages/core/src/browser/opener-service.ts

```diff
@@ -84,12 +88,27 @@ export async function open(openerService: OpenerService, uri: URI, options?: Ope
 
 @injectable()
 export class DefaultOpenerService implements OpenerService {
+    // Containing OpenHandlers created for CustomEditor contributions
+    protected readonly additionalHandlers: OpenHandler[] = [];
```

Comment:
Based on the documentation would the following variable name be more descriptive?

```suggestion
    // Collection of open-handlers for custom-editor contributions.
    protected readonly customEditorOpenHandlers: OpenHandler[] = [];
```

Reply:
Updated as requested. The thought here was maybe it will be used for other uses in future so therefore name was more generic.